src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -220,7 +233,11 @@ public boolean makeIsland(Player player, String name, Schematics.SchematicConfig
 
         player.sendMessage(StringUtils.color(IridiumSkyblock.getInstance().getMessages().creatingIsland.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
         createIsland(player, name, schematicConfig).thenAccept(island -> {
-                    player.teleport(island.getHome());
+                    if (!PaperLib.isPaper()) {
```

Comment:
you don't need to check env, iikrc, paper is already checks it

Reply:
Paper checks it? how?

Reply:
PaperLib#initialize checks envs and just uses CompletableFuture so paper or spigot isn't matter?

Reply:
Yeah, PaperLib#teleportAsync should only teleport asynchronously if Paper is installed. I thought you were aware of that and tried to fix a bug

Reply:
The mspt between teleporting spigot and paper is totally different. A real gain